### PR TITLE
Integration test for Spark backend

### PR DESCRIPTION
**Pull request by shravan97**
Features in this PR : 
* Convert the [GSoC tasks code](https://github.com/etejedor/GSoC_2018/blob/master/Spark_DistROOT_TDataFrame.ipynb) into an integration test and add it under Spark integration tests

### CODE REVIEW

File: tests/integration/spark/test_histograms.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        """
+        Cleans up the `SparkContext` objects
+        that were created.
+
+        """
+        from pyspark import SparkContext
+        context = SparkContext.getOrCreate()
+        context.stop()
+
+    def get_pyrdf_graph(self):
```
Comment (etejedor) [inline tests/integration/spark/test_histograms.py]: Maybe `build_pyrdf_graph` instead of get? The rest looks good!

Comment (shravan97) [reply tests/integration/spark/test_histograms.py]: Ok @etejedor . Will fix this. Shall I also do the final rebase after the fix ?

File: tests/integration/spark/test_histograms.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import unittest, PyRDF
+
```
Comment (etejedor) [inline tests/integration/spark/test_histograms.py]: Perhaps we can call this file: `test_inv_mass.py`?

Comment (shravan97) [reply tests/integration/spark/test_histograms.py]: Ok. Sounds good. I'll change this too 👍

File: tests/integration/spark/test_histograms.py (hunk 1 above)
Comment (etejedor) [reply tests/integration/spark/test_histograms.py]: Yes, you can keep the two commits you have now.

**Comment by shravan97** [comment]
@etejedor , done resolving your comments 👍

**Pull request merged**